finance-tracker/config/routes.rb

```diff
@@ -2,4 +2,6 @@
   devise_for :users
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
   root 'welcome#index'
+  get 'my_portfolio', to: 'users#my_portfolio'
+  get 'search_stock', to: 'stocks#search'
```

Comment:
use `resources`

Reply:
but then the actions are not index, delete, new, create etc wor, how to use resources with specific URL and action?

Reply:
ganbatte